Do not install grotty:

Command 'which grotty' returned error. Aborting.

You need to install grotty for the man page feature to work
If you don't care about this then delete the exit command below
in the configure script

--- configure.orig	2026-10-14 21:58:00 UTC
+++ configure
@@ -14061,7 +14061,7 @@ else
     echo "If you don't care about this then delete the exit command below" ;
     echo "in the configure script" ;
     echo "" ;
-    exit 1 ;
+#    exit 1 ;
 fi
 if test "x`uname`" = xFreeBSD ; then
   TST_MAN_CMD='for i in `man -a -w ls` ; do man $i ; echo ; echo --- ; echo ; done' ;
